src/libaktualizr/primary/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+    set(CMAKE_CXX_FLAGS -pthread)
```

Comment:
Does it also work if only added in the root CMakeLists.txt, instead of having to duplicate that in several places?

Reply:
Probably Yes, the global flag should work for all targets, I'll check it...